a10_octavia/api/drivers/flavor_schema.py

```diff
@@ -208,6 +208,21 @@
                     "description": "Secondary nameserver used to contact the GLM or ELM"
                 }
             }
+        },
+        "proxy-server": {
+            "type": "object",
+            "description": "Forward proxy-server configuration details.",
+            "properties": {
+                "proxy_host": {
+                    "type": "string"
+                },
+                "proxy_port": {
```

Comment:
These are json definitions so they need to have a dash instead of underscore. They also need a description

Reply:
I had previous added these for STACK-2888 but removing them to prevent clashing. Should just be able to copy and paste this in

```
                "proxy-host": {
                    "type": "string",
                    "description": "Hostname of proxy server used for requests to GLM"
                },
                "proxy-port": {
                    "type": "string",
                    "description": "Port number used to establish connection with proxy server"
                },
                "proxy-username": {
                    "type": "string",
                    "description": "Username for proxy authentication"
                },
                "proxy-password": {
                    "type": "bool",
                    "description": "Password for proxy authentication"
                },
                "proxy-password": {
                    "type": "string",
                    "description": "Password for proxy authentication"
                },
```

Reply:
Used above provided flavor schema content.